src/main.ts

```diff
@@ -48,6 +48,9 @@ import {
 import { type RedemptionInfo } from "./entities/redemption-info";
 import { type PurchaseResult } from "./entities/purchase-result";
 import { mount } from "svelte";
+import { RenderPaywallParams } from "./entities/render-paywall-params";
```

Comment:
We should also export this from here, so it's available to consumers and generated as part of the API.